### PR TITLE
Feat/shavingbar

### PR DESCRIPTION
**Pull request by nikhedonia**

### CODE REVIEW

**Comment by aronshamash** [comment]
I think having a short feature kickoff would have help here. Component architecture would have been agreed upfront.

File: packages/shavingbar/src/styles/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  text: {
+    marginRight: spacing(1)
+  },
+  bubble: {
+    display: "flex",
+    boxSizing: "border-box",
+    justifyContent: "center",
+    alignItems: "center",
+    borderStyle: "solid",
+    borderColor: "rgb(219, 219, 219)",
```
Comment (nehasri89) [inline packages/shavingbar/src/styles/index.js]: use hexcodes

Comment (jimhunty) [reply packages/shavingbar/src/styles/index.js]: Actually use the styleguide colours

**Comment by craigbilner** [comment]
being deprioritised

**Pull request closed without merging**